components/21-atoms/link/_link.scss

```diff
@@ -50,8 +50,8 @@ a {
 
     &[download] {
       &:not(.no-icon):not(.tag),
-      &[href^="http://"],
-      &[href^="https://"] {
+      &[href^="http://"]:not([href*="gent.be"]),
+      &[href^="https://"]:not([href*="gent.be"]) {
```

Comment:
Zijn we zeker dat een *.gent.be domein altijd als intern gezien moet worden?

Reply:
Dat was de scope van het ticket: "alles dat gent.be bevat". Ik kan ook niet echt op voorbeelden komen waar we dat niet willen, tenzij in zeer grote uitzonderingen waar er mss een parameter met gent.be in zit of dergelijke.

Het alternatief is dat we een lijst toevoegen op al deze plaatsen met de websites die geen externe-linkicoon mogen krijgen. En dat leek me meer bloat in code dan die uitzondering (if at all) die er zouden kunnen zijn.